metadata-ingestion/src/datahub/cli/container_cli.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+import click
+
+from datahub.ingestion.graph.client import get_default_graph
+from datahub.metadata.schema_classes import (
+    GlossaryTermAssociationClass,
+    OwnerClass,
+    OwnershipTypeClass,
+    TagAssociationClass,
+)
+from datahub.specific.dataset import DatasetPatchBuilder
+from datahub.utilities.urns.urn import guess_entity_type
+
+logger = logging.getLogger(__name__)
+
+
+@click.group()
+def container() -> None:
+    """A group of commands to interact with containers in DataHub."""
+    pass
+
+
+def apply_association_to_container(
+    container_urn: str,
+    association_urn: str,
+    association_type: str,
+) -> None:
+    """
+    Common function to apply either tags, terms, or owners to containers and their datasets.
+
+    Args:
+        container_urn: The URN of the container
+        association_urn: The URN of the tag, term, or user to apply
+        association_type: One of 'tag', 'term', or 'owner'
+    """
+    urns = [container_urn]
```

Comment:
if we are only applying to datasets, we should not add the `container_urn`, right?